sys-boot/plymouth/plymouth-24.004.60.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
```

Comment:
nitpick: can add an empty line before `EAPI=8`.. just to be inline with `skel.ebuild`